railties/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-*   Make :null_store the default store in the test environment.
+*   Make `ActiveSupport::Cache::NullStore` the default cache store in the test environment.
```

Comment:
`:null_store` isn't clear enough for us? I feel this like a cosmetic change.

Reply:
I think `:null_store` could be, but from "the default store in the test environment." not clear about what kind of store we are talking about.

Reply:
Okey I'll accept "not clear about what kind of store we are talking about" this time.

But we generally do not accept pull request only changing the changelog. https://github.com/rails/rails/pull/32669#issuecomment-383125549

Reply:
I still more lean to `ActiveSupport::Cache::NullStore` in order to reflect our docs https://edgeguides.rubyonrails.org/caching_with_rails.html#activesupport-cache-nullstore.

> But we generally do not accept pull request only changing the changelog. #32669 (comment)

As I remember @rafaelfranca wanted me to say that it was annoying(I admitted that it was annoying) that I was doing those cleanups often. It is better to do that during preparing "Ruby on Rails 6.0 Release Notes" guide.
I'm not sure that we don't allow contributors to fix the changelogs since it is related to documentation. If we don't allow fix the changelogs we should add the mention about it to the guides, but I'm not sure that it is a good idea.
The main reason why I was doing those cleanups of the changelogs is that once it took me an extra few hours after I [prepared Ruby on Rails 5.2 Release Notes](https://github.com/rails/rails/commit/7fd6d69af2f5fcea70c5c7df7d75e547ef9c807e) because [I had to remove entries that backported to 5.1 from changelogs and from Ruby on Rails 5.2 Release Note guide](https://github.com/rails/rails/pull/32281).
Anyway I don't want to violate our rules. Please feel free to close this Pull Request if it violates our rules.